daedalus/src/Daedalus/ClientApi.purs

```diff
@@ -102,6 +102,24 @@ newWallet :: forall eff . EffFn5 (ajax :: AJAX, crypto :: Crypto.CRYPTO | eff) S
 newWallet = mkEffFn5 \wSetName wsAssurance wsUnit mnemonic spendingPassword -> fromAff <<< map encodeJson <<<
     either throwError (B.newWallet $ mkCPassPhrase spendingPassword) $ mkCWalletInit wSetName wsAssurance wsUnit mnemonic
 
+-- | Get meta information from given wallet
```

Comment:
@Martoon-00 we will remove the docs - and long term we will depricate client-api as well as frontend won't use it any more.